cmd/image.go

```diff
@@ -84,7 +91,10 @@ func createCollage(albums []Album, rows int, columns int, displayOptions Display
 
 	dc := gg.NewContext(imageCoverWidth*columns, imageCoverHeight*rows)
 	dc.SetRGB(0, 0, 0)
-	dc.LoadFontFace(fontfile, 12)
+	err := dc.LoadFontFace(fontFile, 12)
+	if err != nil {
```

Comment:
We'll have a figure out a way to handle all our errors, I think it'll be better to return a 500 to the user rather than crashing the server.

Reply:
Yeah agree. Though in this case I think it's fine since if it can't load a font font it's probably missing or corrupt